### PR TITLE
Removes the need to have any code in the iframe

### PR DESCRIPTION
**Pull request by baconface**

### CODE REVIEW

File: public/scripts/app.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    sandbox.api = api;
+
+    // Convert api.move() to move() etc
+    for (key in api) {
+      sandbox[key] = api[key];
+    }
+
+    // Prevent access to parent objects and their methods
+    sandbox.api = "";
+    sandbox.window.parent = "";
```
Comment (blainesch) [inline public/scripts/app.js]: Tricky. I wonder if this would break anything.

Comment (baconface) [inline public/scripts/app.js]: Break what exactly? Yeah, it looks pretty creepy and you would think it would. But all seemed to work fine on me end.

Comment (baconface) [inline public/scripts/app.js]: If you mean how it uses a bridge after it burned it you're right it does come off as code smell. Works for me in Chrome, Firefox, and NW.js though. If you have an alternative you feel better about that does the same way I am welcome to it. :smiley:

Comment (blainesch) [inline public/scripts/app.js]: If it works it works! :D

Maybe something like:

~~~ js
// window scope
(function() {
  var parent = undefined,
    window = undefined;
  // execute code here
})();
~~~

They won't have access to `window` or `parent` but it doesn't remove it completely.

**Pull request merged**